tink-docker/main.go

```diff
@@ -103,7 +101,7 @@ func rebootWatch() {
 	// Forever loop
 	for {
 		if fileExists("/worker/reboot") {
-			//Build the command, and execute
+			// Build the command, and execute
```

Comment:
Remove this extraneous comment. It only describes what the trivial code below does, rather than the why.

Reply:
> Remove this extraneous comment. It only describes what the trivial code below does, rather than the why.

I can fix this but in a lot of places, I see similar comments. example: create directory,  download configuration...

Reply:
Let's just fix the ones we run into in the course of fixing other items. 

> "a comment should only be added to answer a question that the code can’t."
> -- https://itnext.io/what-makes-a-good-code-comment-5267debd2c24

Reply:
Fixed